src/include/switch_core_media.h

```diff
@@ -102,6 +106,7 @@ typedef enum {
 
 typedef struct switch_core_media_params_s {
 	uint32_t rtp_timeout_sec;
+	uint32_t partner_rtp_timeout_sec;
```

Comment:
Use a different variable for rtp media timeout so it wouldn't conflict with rtp_timeout_sec